pom.xml

```diff
@@ -43,7 +42,6 @@
 		<module>driver-nats-streaming</module>
 		<module>driver-nsq</module>
 		<module>driver-jms</module>
-		<module>driver-redis</module>
```

Comment:
Pravega and redid shouldn't be removed from here

Reply:
Fixed